src/Instances.js

```diff
@@ -67,8 +67,8 @@ const searchableIndexes = [
   { label: 'ui-inventory.instanceId', value: 'id', queryTemplate: 'id="%{query.query}"' },
   { label: 'ui-inventory.title', value: 'title', queryTemplate: 'title="%{query.query}"' },
   { label: 'ui-inventory.identifier', value: 'identifier', queryTemplate: 'identifiers adj "\\"value\\": \\"%{query.query}\\""' },
-  { label: 'ui-inventory.search.isbn', value: 'isbn', queryTemplate: 'identifiers == "*\\"value\\": \\"%{query.query}\\", \\"identifierTypeId\\": \\"<%= identifierTypeId %>\\""' },
-  { label: 'ui-inventory.search.issn', value: 'issn', queryTemplate: 'identifiers == "*\\"value\\": \\"%{query.query}\\", \\"identifierTypeId\\": \\"<%= identifierTypeId %>\\""' },
+  { label: 'ui-inventory.isbn', isSubIndex: true, value: 'isbn', queryTemplate: 'identifiers == "*\\"value\\": \\"%{query.query}\\", \\"identifierTypeId\\": \\"<%= identifierTypeId %>\\""' },
```

Comment:
What about adding prefix field instead?

`'prefix': '-'`
and then in the code check for its existence and add it.

Reply:
@VictorSoroka1 thanks for the great solution!